src/averageAges.js

```diff
@@ -55,7 +89,28 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  function findMothers(child) {
+    return people.find(mother => mother.name === child.mother);
+  }
```

Comment:
does `find` return an array of items or just a single one? 
```suggestion
  function findMother(child) {
    return people.find(mother => mother.name === child.mother);
  }
```